lib/ansible/modules/cloud/google/drive_append_to_sheet.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2018, Chris Blum <chris.n.blum@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'], 'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: drive_append_to_sheet
+
+short_description: Archive variables in a Google Sheet
+
+version_added: "2.8"
+
+description:
+    - "Using this module it is possible to archive variables in a playbook run into a Google Sheet.
+      This is especially interesting if you are trying to gather facts across lots of servers and
+      you will want to do some statistical analysis on that data later on.
+      A Google account and the Python client library for Google APIs are requirements.
```

Comment:
The python library should go into `requirements:`. The Google account is actually related to a parameter, so that information should go with the required parameter.

Reply:
I think what you meant was already in there - but I removed it from the description...